### ISSUE TITLE
unstable scaling test?

### ISSUE THREAD

**Issue by Anthchirp** [xia2/xia2#341]
```
Test.Modules.Scaler.test_DialsScalerHelper.test_dials_symmetry_indexer_jiffy[refiner_lattices2-expected_output2] (from pytest) 

AssertionError: assert ('P 1', False... False, False) == ('P 1', False,..., False, True)   At index 4 diff: False != True   Full diff:   - ('P 1', False, False, False, False)   ?                              ^^^^   + ('P 1', False, False, False, True)   ?                              ^^^

Stacktrace

helper = <xia2.Modules.Scaler.DialsScaler.DialsScalerHelper object at 0x7f6880283ad0>
refiner_lattices = ['aP'], expected_output = ('P 1', False, False, False, True)

    @pytest.mark.parametrize("refiner_lattices, expected_output", test_lattices)
    def test_dials_symmetry_indexer_jiffy(helper, refiner_lattices, expected_output):
        """Test the jiffy"""
        n = 2
        if n > 1:
            multisweep = True
        # Create list of experiments, reflections and refiners
        experiments = []
        reflections = []
        refiners = []
        for i in range(0, n):
            refl_path, exp_path = ("test_%s.refl" % i, "test_%s.expt" % i)
            generate_reflections_in_sg("P 2", id_=i, assign_id=True).as_pickle(refl_path)
            generated_exp(space_group="P 2", id_=i).as_file(exp_path)
            experiments.append(exp_path)
            reflections.append(refl_path)
            refiners.append(simple_refiner(refiner_lattices))
    
        result = helper.dials_symmetry_indexer_jiffy(
            experiments, reflections, refiners, multisweep=multisweep
        )
        pg, reind_op, ntr, pt, reind_refl, reind_exp, reind_init = result
        refiner_reset = refiners[0].get_refiner_reset()
>       assert (pg, ntr, pt, refiner_reset, reind_init) == expected_output
E       AssertionError: assert ('P 1', False... False, False) == ('P 1', False,..., False, True)
E         At index 4 diff: False != True
E         Full diff:
E         - ('P 1', False, False, False, False)
E         ?                              ^^^^
E         + ('P 1', False, False, False, True)
E         ?                              ^^^

../../../xia2/Test/Modules/Scaler/test_DialsScalerHelper.py:390: AssertionError
```

**Comment by jbeilstenedmands** [comment]
Based on the stracktrace, this is from before I 'fixed' a few things this morning.

I've been looking into the current version of the code and I can't reproduce on linux or mac. I've looked at the code and am happy that it's doing what I think and that the expected result is correct, so I'm not sure what else to do for this but wait for the next build and hope its ok.

**Comment by Anthchirp** [comment]
verified this was before commits
```
Git (git git@github.com:dials/cctbx.git)

    fix repository location (commit: c956368) (detail)

Git (git git@github.com:dials/dxtbx.git)

    Don't break (commit: 1bbe863) (detail)
    Add test for dials/dials#979 (commit: 869b2b2) (detail)

Git (git git@github.com:xia2/xia2.git)

    Update code for merging of space group checking into dials.symmetry (commit: f8b408c) (detail)
    Fix DialsScaler test. Also remove unused function (commit: e404b65) (detail)
    Refactor tests to avoid chdir() (commit: ef89392) (detail)
    Mark XDS tests as skipped if no XDS installed (commit: 66f2318) (detail)
    Revert 66f2318c0e1a8442415ea86c26ed5552fc16d1fd (commit: a9d1952) (detail)
    Mark more XDS tests as skipped if no XDS installed (commit: 1e47cfb) (detail)
    deprecate Applications/xia2merger.py (commit: 21f49fe) (detail)

Git (git git@github.com:dials/dials.git)

    Document expected exceptions from run_dials_refine (commit: 0b1ee6e) (detail)
    Use imageset_id from input reflections if present (commit: 2e28954) (detail)
    Not Sorry (commit: fb0f9be) (detail)
    Document expected exceptions from indexing (commit: 6d8503d) (detail)
    Remove unused variables from GModelBackgroundCreator (#974) (commit: b6167d8) (detail)
    Add systematic absences checking for dials.symmetry. (#951) (commit: 9780dfd) (detail)
    Fix documentation build errors following #951 (commit: 6a31f50) (detail)
```

Running build again now.

**Comment by Anthchirp** [comment]
looking good
